internal/host/transition.go

```diff
@@ -29,7 +29,7 @@ type transitionHandler struct {
 	eventsHandler eventsapi.Handler
 }
 
-var resetFields = [...]interface{}{"inventory", "", "bootstrap", false}
+var resetFields = [...]interface{}{"inventory", "", "bootstrap", false, "progress_current_stage", "", "progress_installation_percentage", 0}
```

Comment:
The **cluster** progress bar consist of 3 parts:
* prepare for installation (fist 10%)
* hosts installation
* operators installation

We need to make sure we reset:
1. the host progress
2. the cluster progress as well

Reply:
Added Missing fields on host, cluster fields and a subsystem test